tests/tabular/checks/model_evaluation/simple_model_comparison_test.py

```diff
@@ -68,6 +68,18 @@ def test_classification_binary_string_labels(iris_binary_string_split_dataset_an
     assert_classification(result, ['a', 'b'])
 
 
+def test_classification_binary_string_labels_custom_scorer(iris_binary_string_split_dataset_and_model):
+    # Arrange
+    train_ds, test_ds, clf = iris_binary_string_split_dataset_and_model
+    check = SimpleModelComparison(scorers=[get_scorer('f1')])
+    # Act X
+    result = check.run(train_ds, test_ds, clf).value
+    # Assert
+    assert_that(result, equal_to({'scorers_perfect': {'f1_score': 1.0},
```

Comment:
If can be done easily, remove from the result.value the redundant dict, i.e make it 
'f1_score': {'Origin': 0.9411764705882353, 'Simple': 0.0}}

Reply:
might destroy stuff